script.js

```diff
@@ -0,0 +1,74 @@
+
+// 카운트 하기 위해 할 일들 목록 각각 가져오기
+let waitingNum = document.getElementsByClassName("waiting");
+let completeNum = document.getElementsByClassName("compted");
+
+// 할 일 추가
+function addList (e) {
+    if (e == 'click' || e.keyCode == 13) {
+        if (document.getElementById("add_input").value != '') {
+            let todo = document.getElementById('add_input').value;
+
+            // 껍데기 div부터 차례로 생성.
+            let listbox = document.createElement('div');
+            listbox.className = 'listbox';
+
+            // li 생성.
+            let list = document.createElement('li');
+            list.className = "waiting";
```

Comment:
`className`을 직접 설정해줘도 좋지만, `classList.add()`를 사용해보면 더 좋을 것 같아요. ([링크](https://www.geeksforgeeks.org/javascript-adding-a-class-name-to-the-element/))

```suggestion
            list.classList.add('waiting');
```

Reply:
classList.add 라는 것이 있었군요...
알려주셔서 감사합니다 ㅜㅜ